docs/setup-aws-lambda.asciidoc

```diff
@@ -1,85 +1,62 @@
 [[aws-lambda]]
-=== Get started with Lambda
+=== Monitoring AWS Lambda Java Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
+The Java APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
-NOTE: In order to get the full AWS Lambda tracing capabilities, use with APM Server 7.16 or higher. Using
-older versions provides most value, but some relevant metadata fields will not be indexed.
+Follow the <<aws-lambda-java-quick-start>> Guide to get started in minutes and checkout the <<aws-lambda-features-and-caveats>> to learn about the features and limitations of the Java APM Agent on AWS Lambda Functions.
 
 [float]
-[[aws-lambda-runtimes]]
-==== Supported runtimes
+[[aws-lambda-java-quick-start]]
+==== Quick Start
 
-AWS Lambda provides multiple https://docs.aws.amazon.com/lambda/latest/dg/java-image.html[JVM base images], only the ones that support the `AWS_LAMBDA_EXEC_WRAPPER` environment variables
-are supported out of the box.
+include::{apm-aws-lambda-root}/docs/add-extension/code.asciidoc[]
 
-Running with unsupported images is still possible but requires providing agent configuration through environment variables
-explicitly.
+Before starting with the setup make sure you are using one of the  <<supported-aws-lambda-runtimes, supported AWS Lambda runtimes>>.
 
-|===
-|Tags |Java Runtime |Operating System|Supported
-
-|11
-|Java 11 (Corretto)
-|Amazon Linux 2
-|yes
-
-|8.al2
-|Java 8 (Corretto)
-|Amazon Linux 2
-|yes
-
-|8
-|Java 8 (OpenJDK)
-|Amazon Linux 2018.03
-|no
-
-|===
+To get started with APM for your Java AWS Lambda functions, follow the steps below.
 
 [float]
-[[aws-lambda-installation]]
-==== Installation
-
-Setting up APM for your Java Lambda function is a two-step process. Make sure to follow _both_ of the following steps:
+===== Step 1: Select the AWS Region and Architecture
 
-1. <<aws-lambda-extension>>
-2. <<aws-lambda-instrumenting>>
+include::{apm-aws-lambda-root}/docs/lambda-selector/lambda-attributes-selector.asciidoc[]
```

Comment:
<img width="759" alt="Screen Shot 2022-03-29 at 3 27 05 PM" src="https://user-images.githubusercontent.com/5618806/160717199-a5c8b667-5fa3-4667-a79e-ca9ff4052ffb.png">

I thought this was broken at first because I didn't immediately see what these selectors edited. I wonder if we should update this text to say something like:

> Select the AWS region and architecture of your Lambda function. This documentation will update based on your selections.

Reply:
The source is in the apm-aws-lambda repo. I fixed the above in https://github.com/elastic/apm-aws-lambda/pull/158